tests/envcheck/envcheck.go

```diff
@@ -0,0 +1,327 @@
+/*
```

Comment:
Most of the time the doc strings are written as single line comments:

```
// first
// second
// ...
```
Possibly a bit annoying, but it is consistent with the majority of the code.

Reply:
Fixed.  Added copyright notice too.